src/VisualStudio/Core/Def/LanguageService/AbstractLanguageService`2.IVsImmediateStatementCompletion2.cs

```diff
@@ -50,8 +51,10 @@ int IVsImmediateStatementCompletion2.InstallStatementCompletion(int install, IVs
         {
             if (!this.filters.TryGetValue(textView, out var filter))
             {
+                var editorAdaptersFactoryService = this.Package.ComponentModel.GetService<IVsEditorAdaptersFactoryService>();
```

Comment:
Rather than having any specific helper for this, I'm just inlining a fetch of the service when we need it. There's no reason to abstract any of this when the line of code is straightforward as it is.